src/sentry/rules/filters/latest_adopted_release_filter.py

```diff
@@ -185,8 +187,8 @@ def clear_get_first_last_release_for_group_cache(instance: GroupRelease, **kwarg
     )
 
 
-def get_project_release_cache_key(project_id: int, environment_id: int) -> str:
-    return f"project:{project_id}:env:{environment_id}:latest_release_adopted"
+def get_project_release_cache_key(project_id: int, environment: str) -> str:
+    return f"project:{project_id}:env:{environment}:latest_release_adopted"
```

Comment:
@wedamija why are we changing the cache key to the name from the id?

Reply:
I changed the parameter to accept the env name for consistency with other related endpoints and so just changed this as well. It's breaking tests though, so I'll rework this.